CHANGELOG.rst

```diff
@@ -33,6 +33,53 @@ Change history
     For detailed configuration, see :ref:`Admin OIDC Configuration Step  <ref_step_mozilla_django_oidc_db.setup_configuration.steps.AdminOIDCConfigurationStep>`.
     Make sure to check which fields are marked as ``DEPRECATED`` and replace them with the fields that are mentioned as replacements.
 
+**New features**
+
+* [maykinmedia/open-objects#18] Add booleanfield that indicates whether his object should be linkable to zaken
+* [maykinmedia/open-api-framework#188] Add csv option to data dump script
+* [maykinmedia/open-api-framework#152] Add OpenTelemetry (``OTel``) for collecting and exporting application metrics.
+
+.. note::
+
+    - Metrics now exposed include:
+
+        - HTTP request durations.
+        - Number of active requests.
+        - Number of users, logins, logouts, failed logins, and account lockouts.
+        - **CRUD** operations for the following endpoint:
+
+          - ``/objecttype``
+
+        - All metrics are exported using the **OpenTelemetry standard**, enabling seamless integration with existing monitoring and visualization platforms.
+
+.. warning::
+
+    The OpenTelemetry SDK is **enabled by default**.
+
+    If you do not have an endpoint to send system telemetry to, update your deployment to **disable it** by setting the environment variable:
+
+    .. code-block:: bash
+      
+        OTEL_SDK_DISABLED=true
+        
+    If this is not done, warnings will be emitted to the container logs. The application will continue to function normally.
+
+    All available metrics and details can be found in the **“Observability” documentation** see :ref:`installation_observability_index`
```

Comment:
Let's make this a note instead of a warning, since leaving it on doesn't break the application
```suggestion
.. note::

    The OpenTelemetry SDK is **enabled by default**.

    If you do not have an endpoint to send system telemetry to, update your deployment to **disable it** by setting the environment variable:

    .. code-block:: bash

        OTEL_SDK_DISABLED=true

    If this is not done, warnings will be emitted to the container logs. The application will continue to function normally.

    All available metrics and details can be found in the :ref:`Observability documentation <installation_observability_index>`.
```